iotileship/iotile/ship/actions/ModifyJsonStep.py

```diff
@@ -0,0 +1,89 @@
+import json
+import copy
+
+from iotile.ship.exceptions import RecipeActionMissingParameter
+
+def modify_dict(data, key, value, create_if_missing=False):
+    """
```

Comment:
Style: comment blocks are supposed to start on the same line as the opening quotes (throughout)